apps/discord-bot-frontend/src/lib/discord/client.ts

```diff
@@ -424,6 +424,48 @@ client.on(Events.ThreadUpdate, async (oldThread, newThread) => {
     console.debug('[client:events:ThreadUpdate] finished')
   }
 })
+client.on(Events.ThreadDelete, async(thread) => {
+  console.debug('[client:events: ThreadDelete] Thread Deleted started');
+
+  if(!thread.id){
```

Comment:
can you run this through prettier? the `check` command will pick up differences, https://github.com/aws-amplify/discord-bot/blob/main/package.json#L13